content/news/042/index.md

```diff
@@ -209,6 +209,32 @@ December's full weekly devlogs: "This Week In Veloren...":
 
 ## Engine Updates
 
+### [Fyrox]
+
+![Particle System Preview](particle_system.gif)
+
+[Fyrox] ([Discord][fyrox_discord], [Twitter][fyrox_twitter]) is a game engine that
```

Comment:
Maybe it's worth adding a direct link to https://fyrox.rs/blog/post/feature-highlights-0-29/ somewhere?